cmd/keymasterd/config.go

```diff
@@ -448,6 +452,31 @@ func loadVerifyConfigFile(configFilename string,
 		return nil, err
 	}
 
+	if hasDNS, err := runtimeState.Config.DnsLoadBalancer.HasDNS(); err != nil {
```

Comment:
rename this.... hasDNS is too generic... hasDNSLb...

Reply:
Done.